wwwapp/settings.py

```diff
@@ -159,7 +162,33 @@
 
 AUTHENTICATION_BACKENDS = (
     'django.contrib.auth.backends.ModelBackend',
-    'allaccess.backends.AuthorizedServiceBackend',
+    'social_core.backends.google.GoogleOAuth2',
+    'social_core.backends.facebook.FacebookOAuth2',
+)
+
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = os.environ['SOCIAL_AUTH_GOOGLE_OAUTH2_KEY']
```

Comment:
Add a check if this is actually defined (for development) and make it fail gracefully if not (warning in console + remove from AUTHENTICATION_BACKENDS)